zulipterminal/model.py

```diff
@@ -1144,6 +1147,14 @@ def user_name_from_id(self, user_id: int) -> str:
 
         return self.user_dict[user_email]["full_name"]
 
+    def user_name_count(self, user_name: str) -> int:
+        """
+        Returns the count of the users with the same name as the given name.
+        """
+        if not self.user_name_dict_count[user_name]:
+            return -1
+        return self.user_name_dict_count[user_name]
```

Comment:
I think it would be better/safer if you used `.get()` to access the count, instead of using the index.